src/VisualStudio/Xaml/Impl/Microsoft.VisualStudio.LanguageServices.Xaml.csproj

```diff
@@ -9,7 +9,6 @@
     <DeployExtension>false</DeployExtension>
     <TargetFramework>net46</TargetFramework>
     <ResolveAssemblyWarnOrErrorOnTargetArchitectureMismatch>None</ResolveAssemblyWarnOrErrorOnTargetArchitectureMismatch>
-    <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
```

Comment:
Yes, because otherwise package mismatches cause a bunch of extra MSBuild warnings, as it tries to compute a set of binding redirects that will never be used anywhere.